docs/Api/DefaultApi.md

```diff
@@ -120,9 +120,6 @@ Name | Type | Description  | Notes
 - **Content-Type**: `application/json`
 - **Accept**: `application/json`
 
-[[Back to top]](#) [[Back to API list]](../../README.md#endpoints)
-[[Back to Model list]](../../README.md#models)
-[[Back to README]](../../README.md)
```

Comment:
why remove these?

Reply:
oh did not meant to, I'll get those back in